docs/features/vpn.rst

```diff
@@ -125,7 +125,7 @@ Configurable Method
 """""""""""""""""""
 
 From the site configuration, fastd can be allowed to offer
-toggleable encryption in the config mode with the intent to
+togglable encryption in the config mode with the intent to
```

Comment:
Toggleable/togglable is not a word with formal spelling. Once that changes, we can swap them; for now they are both equally right.

```suggestion
togglable encryption in the config mode with the intent to
```

Reply:
```suggestion
encryption which can be toggled in the config mode with the intent to
```

Reply:
I feel like this already is precise, does not need changing and therefore should not; but maybe others differ.